src/pages/post-feed/index.tsx

```diff
@@ -0,0 +1,111 @@
+import Image from 'next/image';
+import CreatPost from './CreatPost';
```

Comment:
this component is not available in this commit, maybe you forgot to commit it

it is fine to leave this for now, but next time make sure that each commit you do is executable

Reply:
Did you mean here I shouldn't commit these two lines? 
if yes, can you explain to me why shouldn't? please.

Reply:
because if we checkout to this commit and try to run the app it will crash, the error would be `cannot resolve CreatePost` or something like that because this component is not defined yet